libavfilter/dnn/libtf_wrapper.h

```diff
@@ -0,0 +1,683 @@
+/*
```

Comment:
Is it possible to move this header into `compat/dnn` instead of `libavfilter/dnn`? Would be similar to where other shared library wrapper reside, example CUDA.

Reply:
Moved to compat/dnn